tools.yml

```diff
@@ -293,6 +305,8 @@ tools:
     mem: 12
   toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_multijoin_tool/.*:
     mem: 8
+  toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sort_header_tool/.*:
+    mem: 10
```

Comment:
That looks like way too much for the general case. The tool is not doing much afaik.
Maybe we should include here a rule?

Reply:
Thanks very much for reviewing! This rarely uses above the default 3.8GB and only when the inputs are large, so I'll add a rule here.